### DISCUSSION TITLE
How to prevent line breaks

### DISCUSSION THREAD

**Discussion by starxyn** [mantinedev/mantine#6874] [Q&A] [answered]
`<Grid> <Grid.Col span={12}> {show && ( <Text c='gray.9'>ABC</Text> <Text c='red.7'>*</Text> ) </Grid.Col> </Grid> `

The result is:
ABC
*
which I don't want cos I want it to be ABC* and not having a line break in between ABC and *.

**Comment by Kenzo-Wada** [comment]
Can you provide the entier code of yours?

**Reply by starxyn** [reply]
Updated as above. Sorry some parts were hidden as I didn't wrap it in markdown.

**Reply by Kenzo-Wada** [reply, marked as answer]
@starxyn 
Ok, Now I see where the problem is!

Inside `<Grid.Col>` is not a Flex Box, so you have to wrap those two text with it.

eg.
```js
<Grid>
  <Grid.Col span={12}>
    {show && (
    <Flex gap={0}> // or <Group>
      <Text c='gray.9'>ABC</Text>
      <Text c='red.7'>*</Text>
    </Flex>
  </Grid.Col>
</Grid>
```

**Reply by starxyn** [reply]
this works. thanks!
